index.d.ts

```diff
@@ -1,15 +1,16 @@
 
-declare class Nuid {
+export const VERSION: string;
 
-    /**
-     * Resets the prefix of the global nuid, as well as the
-     * pseudo random sequence number and increment amounts.
-     */
-	next():string;
+/** Returns the next nuid from the global generator.
+ *  
+ * @export
+ * @returns {string}
+ */
+export declare function next(): string;
 
-    /**
-     * Resets the prefix of the global nuid from random bytes, as
-     * well as the pseudo random sequence number and increment amounts.
-     */
-    reset():void;
-}
\ No newline at end of file
+/**
+ * Resets the prefix of the global nuid from random bytes, as
+ * well as the pseudo random sequence number and increment amounts.
+ * @export
+ */
+export declare function reset(): void
```

Comment:
As I understand it `declare` is used for instructing typescript that a variable is defined elsewhere. Here' we are just exporting a function.